docs/cvl/methods.md

```diff
@@ -346,8 +346,18 @@ determine whether the call should be replaced by an approximation.
 To determine whether a function call is replaced by an approximation, the
 Prover considers the context in which the function is called in addition to the
 application policy for its signature.  If present, the application policy must
-be either `ALL`, `UNRESOLVED`, or `DELETE`; the default policy is `ALL` with the exception
-of `DISPATCHER` summaries, which have a default of `UNRESOLVED`.
+be either `ALL`, `UNRESOLVED`, or `DELETE`.
+The `ALL` policy indicates the summary should be applied to all instances of the 
+specified method, while `UNRESOLVED` applies only to methods that cannot be fully
+resolved (i.e., both target contract and method identifier are known).
+For internal summaries, the default is `ALL`, as all internal functions
+are always resolvable; thus `UNRESOLVED` is impossible and will yield an error. 
+Similarly, for external summaries with contract-specific entries, 
+the default policy is `ALL`. 
+Conversely, for any external summary on wildcard contracts, the default 
+policy is `UNRESOLVED`. One may apply the `ALL` policy to make the summary apply
+on all instances of the wildcard method, even on target contracts for which
+it was resolved, e.g. by {ref}`linking <--link>`.
```

Comment:
I think this whole paragraph is quite confusing.

I think the confusion has to do with the "context" and "resolved" parts not really being explained. (also couldn't find it further up the doc)

So, instead of talking about "context in which the function is called" abstractly, those contexts should be named.

If things don't fit one paragraph, it's ok to forward-reference like that in an overview-paragraph, but then a paragraph explaining "context" and "resolved" business should follow. Then I think this new paragraph will make more sense.

Reply:
added explanation on contexts, on attributes defining a call site, and what it means to resolve a call in a particular call site

Reply:
I think it's much clearer now! -- hopefully for others as well ..